Controller/DrinksInfoController.cs

```diff
@@ -0,0 +1,122 @@
+using drinks_info_console.APIConsumerServices;
+using drinks_info_console.Input;
+using drinks_info_console.Models;
+using drinks_info_console.UI;
+
+namespace drinks_info_console.Controller;
+
+public static class DrinksInfoController
+{
+    private static UserInput UserInput = new UserInput();
+    private static TableBuilder TableBuilder = new TableBuilder();
+    private static Validation Validator = new Validation();
+
+    public static async Task StartProgram()
+    {
+        string ans, categoryName, drinkId;
+        List<DrinkDetail> drinkDetails;
+        List<object> cleanDrinkDetails;
+
+        do
+        {
+            categoryName = await SelectCategoryAsync();
+            drinkId = await SelectDrinkAsync(categoryName);
+
+            drinkDetails = await DrinksProcessor.GetDrinksDetailsAsync(drinkId);
+            cleanDrinkDetails = GetCleanDetailsList(drinkDetails);
+
+            TableBuilder.DisplayTable(cleanDrinkDetails, "");
+
+            Console.Write("Do you want to check another drink? (y/n): ");
+            ans = UserInput.GetInput().ToLower();
+            while (!Validator.IsValidAns(ans))
+            {
+                Console.WriteLine("Invalid Input!");
+                Console.Write("Do you want to check another drink? (y/n): ");
+                ans = UserInput.GetInput().ToLower();
+            }
+        } while (ans == "y");
+    }
+
+    private static async Task<string> SelectCategoryAsync()
+    {
+        var categoryList = await DrinksProcessor.GetCategoriesListAsync();
+
+        TableBuilder.DisplayTable(categoryList, "Category");
+
+        var categoryChoice = GetValidCategoryChoice();
+        while (!Validator.IsValidCategory(categoryList, categoryChoice))
+        {
+            Console.WriteLine("Invalid Category!");
+            categoryChoice = GetValidCategoryChoice();
+        }
+
+        return categoryChoice;
+    }
+
+    private static string GetValidCategoryChoice()
+    {
+        Console.Write("Enter a category name to select: ");
```

Comment:
I found an issue with the user input. When I try a multi-word choice it says my selection is invalid: 

![image](https://user-images.githubusercontent.com/108711476/205421812-0b1f0cfc-03bc-43ea-a0b1-ab67113cd182.png)

Reply:
I don't know how I missed that but I will investigate right away!

Reply:
Oh I see... the program is case sensitive lol. I will fix this

Reply:
done! I will create a pull request in the new repo